src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
move it to `*` tag

Reply:
[CODE KNOWLEDGE] - Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document

Reply:
u can use * selector only with box-sizing, so lets put box-sizing there

Reply:
okey i'm understand thank you